### PR TITLE
[JENKINS-68485] User status icon is misleading

### PR DESCRIPTION
**Pull request by Langer0416**
See [JENKINS-68485](https://issues.jenkins.io/browse/JENKINS-68485).

Before
Before

After
After

### Proposed changelog entries

Replace the status icon in the user sidepanel.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [X] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] New deprecations are annotated with `@Deprecated(since="TODO")` or `@Deprecated(since="TODO", forRemoval=true)` if applicable.
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/6726"></a>

### CODE REVIEW

**Review by MarkEWaite — APPROVED** [review]

**Review by NotMyFault — APPROVED** [review]

**Comment by daniel-beck** [comment]
FWIW [on ci.j.io / 2.346.1, the job "Status" icon is also a magnifying glass](https://ci.jenkins.io/job/Plugins/job/matrix-auth-plugin/job/master/), but [weekly.ci.j.io / 2.357 has a report/document](https://weekly.ci.jenkins.io/job/Demo/) icon there -- so this is not introducing an inconsistency (as I first thought).

**Comment by NotMyFault** [comment]
> so this is not introducing an inconsistency

Yeah, double checked with build status before. This text appears in different sidepanels with different icons quite some time.

---

This PR is now ready for merge. We will merge it after ~24 hours if there is no negative feedback.
Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process.
Thanks!

**Comment by jtnord** [comment]
I think this is a great improvement over what we have before!
Just wondering though why we would use a circle around the "avatar" though - none of our other icons are in circles. (e.g. https://fontawesome.com/icons/user?s=light depending on where we are sourcing these icons.)

File: core/src/main/resources/hudson/model/User/sidepanel.jelly
Comment (jtnord) [inline core/src/main/resources/hudson/model/User/sidepanel.jelly:35]: I think using the non circle is a better fit, but I will not die on this hill
```suggestion
      <l:task contextMenu="false" href="${rootURL}/${it.url}/" icon="symbol-person" title="${%Status}"/>
```

https://github.com/jenkinsci/jenkins/blob/bdaf2a535e3f3b70b52608fe7fc4f576222adaf2/war/src/main/resources/images/symbols/person.svg 

vs

https://github.com/jenkinsci/jenkins/blob/master/war/src/main/resources/images/symbols/person-circle.svg

**Review by jtnord — APPROVED** [review]

**Pull request merged**
